CHANGELOG.md

```diff
@@ -12,6 +10,31 @@ to the next level. Puppet will rerelease a supported module under the original
 versioning scheme. For now we are using a number less than 1.0 to show that this
 could have some technical issues and should be treated as a prerelease version.
 
+## 2016-06-20 Release 1.2.5
```

Comment:
I don't think we should be taking in the change log from the community repo e.g. We never released 1.2.4.  I would expect this to appear as a changelog item for our unsupported releases.